modules/admin-ui/src/main/webapp/scripts/modules/events/controllers/eventController.js

```diff
@@ -984,4 +984,18 @@ angular.module('adminNg.controllers')
       cleanupScopeResources();
     });
   }
-]);
+
+])
+.directive('escKey', function() {
```

Comment:
I feel like the directive in this PR and #1067 should be moved into its own file, and then reused, rather than copy and pasted.